File: example/src/main/java/cn/yangchi/yangchi_mall/ExampleActivity.java
package cn.yangchi.yangchi_mall;

import cn.yangchi.chichi_core.activities.ProxyActivity;
import cn.yangchi.chichi_core.delegates.BaseDelegate;

public class ExampleActivity extends ProxyActivity {

    @Override
    public BaseDelegate setRootDelegate() {
        return new ExampleDelegate();
    }
}

File: example/src/main/java/cn/yangchi/yangchi_mall/ExampleActivity.kt
package cn.yangchi.yangchi_mall

import cn.yangchi.chichi_core.activities.ProxyActivity
import cn.yangchi.chichi_core.delegates.BaseDelegate

class ExampleActivity : ProxyActivity() {

    override fun setRootDelegate(): BaseDelegate {
        return ExampleDelegate()
    }

}
